rotkehlchen/data/all_assets.json

```diff
@@ -6703,6 +6703,14 @@
         "symbol": "OCC",
         "type": "ethereum token"
     },
+    "OCEAN": {
+        "ethereum_address": "0x985dd3D42De1e256d09e1c10F112bCCB8015AD41",
+        "ethereum_token_decimals": 18,
+        "name": "OceanToken",
+        "started": 1555993918,
```

Comment:
Hey @TimDaub thanks a lot for the PR. I checked and cryptocompare does have `OCEAN` so the price feed should also work.

The only question I have is about the timestamp. Shouldn't it be `1556037118`? It's `04/23/2019 @ 4:31pm (UTC)` pm, not am.

Reply:
You're right. I fixed it.